data_steward/cdr_cleaner/clean_cdr_engine.py

```diff
@@ -77,28 +92,40 @@ def clean_dataset(project=None, statements=None, data_stage=stage.UNSPECIFIED):
                                      batch=batch)
 
         except (oauth2client.client.HttpAccessTokenRefreshError,
-                googleapiclient.errors.HttpError):
-            LOGGER.exception("FAILED:  Clean rule not executed:\n%s",
-                             rule_query)
+                googleapiclient.errors.HttpError) as exp:
+
+            LOGGER.error(
```

Comment:
I would leave this as an `exception` to give access to the stack trace.   The failure message is going to contain the extra relevant information.  But this did technically fail when the query was issued.

Reply:
Reverted this back to an exception